app/models/schemata.rb

```diff
@@ -19,17 +19,30 @@ class Schemata
   def initialize
```

Comment:
would love if this file change was in another commit stating that you just fixed indentations. took me a while to verify this is all that is going here.

Reply:
Yes ofc this is just a rough draft